src/edfi-performance-test/edfi_performance_test/factories/resources/grading_period.py

```diff
@@ -8,11 +8,10 @@
 from edfi_performance_test.api.client.school import SchoolClient
 from edfi_performance_test.factories.resources.api_factory import APIFactory
 from edfi_performance_test.factories.descriptors.utils import build_descriptor
-from edfi_performance_test.factories.utils import UniquePrimaryKeyAttribute
 
 
 class GradingPeriodFactory(APIFactory):
-    periodSequence = UniquePrimaryKeyAttribute()
+    periodSequence = 1
```

Comment:
king this random was previously causing failures when trying to create other resources that rely on a GradingPeriod. That probably worked fine with the ODS/API because it has been using the Northridge database. For meadowlark, which would probably take 12+ hours to load Northridge at this point, I only pre-loaded a very minimal data set, not including grading periods.

Reply:
I must have copied and pasted the text above into the PR, but missed the beginning of the first sentence. I don't know what I was trying to say at the beginning. Point is, I didn't see any reason to have a randomly-assigned `periodSequence`, and it was making life harder for working with Meadowlark.